reana-ui/src/client.js

```diff
@@ -28,7 +28,9 @@ export const GITLAB_AUTH_URL = `${api}/api/gitlab/connect`;
 export const GITLAB_PROJECTS_URL = `${api}/api/gitlab/projects`;
 export const GITLAB_WEBHOOK_URL = `${api}/api/gitlab/webhook`;
 export const WORKFLOWS_URL = (params) =>
-  `${api}/api/workflows?verbose=true&${stringifyQueryParams(params)}`;
+  `${api}/api/workflows?include_progress=true&include_workspace_size=true&${stringifyQueryParams(
+    params,
+  )}`;
```

Comment:
We should remove the first commit so that this URL goes back to the original one, as otherwise open interactive sessions are currently not shown in the UI